### PR TITLE
enable fillMode to support an nth-column option

### PR DESCRIPTION
**Pull request by chbonser**
https://github.com/Addepar/ember-table/issues/634

screen shot 2018-11-26 at 8 45 35 am

### CODE REVIEW

**Comment by cyril-sf** [comment]
@chbonser sorry for the delay.
It seems like a good option. Could you update your PR

**Comment by chbonser** [comment]
@cyril-sf Yes. I can make those changes. It will be a few weeks though...

**Comment by cyril-sf** [comment]
@chbonser no problem and thanks for the help!

**Comment by billdami** [comment]
would love to see this merged in. I could possibly help out getting this across the finish line if you still don't have the bandwidth to work on it @chbonser ?

**Comment by chbonser** [comment]
Sorry for the delay. I will make time for it next week.

**Comment by chbonser** [comment]
@cyril-sf I've rebased and updated the fill mode implementation to reuse the resize implementation across all single column fill modes. The behavior of the preexisting modes remains unchanged so this is a non-breaking addition.

File: addon/components/ember-thead/component.js [hunk 1]
```diff
@@ -118,12 +118,19 @@ export default Component.extend({
...
     * "last-column": extra space is added into the last column.
+    * "nth-column": extra space is added into the column defined by `fillColumnIndex`.
 
     @argument fillMode
     @type string? ('equal-column')
   */
   fillMode: defaultTo(FILL_MODE.EQUAL_COLUMN),
 
+  /**
+    A configuration that controls how which column shinks (or extends) when `fillMode` is
```
Comment (bantic) [inline addon/components/ember-thead/component.js]:
```suggestion
    A configuration that controls which column shrinks (or extends) when `fillMode` is
```

File: addon/components/ember-thead/component.js [hunk 2]
```diff
@@ -118,12 +118,19 @@ export default Component.extend({
...
+    'nth-column'. This is zero indexed.
+  */
```
Comment (bantic) [inline addon/components/ember-thead/component.js]: For the autogenerated API docs can you add:

```
@argument fillColumnIndex
@type number?
```

**Comment by cyril-sf** [comment]
@chbonser no need to apologize, we appreciate the extra effort!

**Comment by bantic** [comment]
released in `v2.1.0`, thanks!

**Pull request merged**